ledger/ledger-on-sql/BUILD.bazel

```diff
@@ -340,6 +340,25 @@ conformance_test(
     ],
 )
 
+conformance_test(
```

Comment:
@rautenrieth-da can you please verify that all conformance test suites are active, which should be, for H2?